app/_locales/en/messages.json

```diff
@@ -1132,20 +1132,20 @@
     "message": "from",
     "description": "Part of the sentence describing when and where snap was added"
   },
-  "flaskWelcomeUninstall": {
-    "message": "uninstall this extension immediately",
+  "flaskWelcomeDoNotUse": {
+    "message": "we recommend against using this extension",
```

Comment:
Why the softer wording? This is more than just a recommendation. This sounds like we're saying that it might be a bad idea. But no, it's definitely a bad idea.

Reply:
The POV to consider is that you are an uninformed user following the instructions of a trusted support agent who is actually a phisher. I was hoping the strong wording would jolt some sense into them.

Reply:
I don't know that that's true; I can think of a variety of technically competent users that aren't developers but may nevertheless want to use Flask despite the risks. That category describes a significant portion of crypto Twitter, for example.

Reply:
Discussion resolved on Slack, and we found a compromise wording